project_task_reassign/project.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2013 Daniel Reis
```

Comment:
I think copyright year needs an update AFAIK you didn't created a v8.0 module in 2013 :)